docs/react-testing-library/migrate-from-enzyme.mdx

```diff
@@ -233,70 +232,69 @@ test('has correct input value', () => {
+of the query documentation might help you understand the concepts better.
+
+> A `<form>` element must have a `name` attribute in order to have an implicit
+> `role` of `'form'` (as required by the specification).
+
+React Testing Library aims to test the components how users will use them. Users
+see buttons, headings, forms and other elements by their role, not by their
+`id`, `class`, or element tag name. Therefore, when you use React Testing
+Library you should avoid accessing the DOM with the `document.querySelector`
+API. (You _can_ use it in your tests, but it's not recommended for the reasons
+stated in this paragraph.)
+
+React Testing Library exposes some handy query APIs which will help you access
+the component elements efficiently. You can see the
+[list of available queries here](queries/about.mdx#types-of-queries). If you're
+not sure which query you should use in a given situation, we have a great page
+which explains [which query to use](queries/about.mdx#priority), so check it
 out!
 
-If you still have a problem with the React Testing Library's queries, and you're
-not sure which query to use, then check out
-[testing-playground.com](https://testing-playground.com) and the accompanying
-Chrome extension
-**[Testing Playground](https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano/related)**  
-that aims to enable developers to find a better query when writing tests, and it
-helps you find the best queries to select elements. It allows you to inspect the
-element hierarchies in the Chrome Developer Tools, and provides you with
-suggestions on how to select them, while encouraging good testing practices.
-
-## using act() and wrapper.update()
-
-In Enzyme, for some asynchronous purposes, you have to call `act()` to run your
-tests correctly, but in React Testing Library you don't need to use `act()` most
-of the time since it wraps APIs with `act()` so you don't need to manually wrap
-it.
-
-`update()` syncs the Enzyme component tree snapshot with the react component
-tree, often time you might see `wrapper.update()` in Enzyme tests, but React
-Testing Library does not need something like that, good for you since you need
+If you still have a question about which of React Testing Library's queries to
+use, then check out [testing-playground.com](https://testing-playground.com) and
+the accompanying Chrome extension
+**[Testing Playground](https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano)**
+which aims to enable developers to find the best query when writing tests. It
+also helps you find the best queries to select elements. It allows you to
+inspect the element hierarchies in the Chrome Developer Tools and provides you
+with suggestions on how to select them, all while encouraging good testing
+practices.
+
+## Using act() and wrapper.update()
+
+When testing asynchronous code in Enzyme, you usually need to call `act()` to
```

Comment:
Should this section mention the async `findBy`/`waitFor` queries? 
If you agree, feel free to use this PR for it, or create a new PR later.

Reply:
I wouldn't think so. This section is really just pointing out a couple of things you _don't_ need to do when using RTL, not raising awareness of its async features in general.

Reply:
I'm fine with this 👍 thanks for the follow up.